### ISSUE TITLE
Analog API pin_dac_write output cannot drive RGB led

### ISSUE THREAD

**Issue by ancebfer** [wokwi/wokwi-features#482]
**Describe the bug**
After adding a RGB led (common cathode) to custom-chip-analog example, the RGB led never lights for any voltage output.

**To Reproduce**
Modified custom-chip-analog example:
https://wokwi.com/projects/330112801381024338

**Expected behavior**
RGB led light intensity should change proportional to voltage as in rgb-slides example:
https://wokwi.com/projects/306455554559050306

**Environment (please complete the following information):**
 - Debian stable
 - Firefox ESR

**Comment by ancebfer** [comment]
Right modified custom-chip-analog-rgb example link:
https://wokwi.com/projects/350784647934771795

**Comment by urish** [comment]
Yeah, unfortunately, LEDs (all of them, RGB included) do not really support analog inputs at the moment. A workaround is to use another custom chip which would do analog to PWM conversion.

**Comment by ikostan** [comment]
Hi @urish,
Is this why RGB led doesn't work in my [WOKWI project](https://wokwi.com/projects/401004574253872129) ?
I tested same code on real device (PiPico W) and it runs just fine...
